src/core/ext/filters/client_channel/lb_policy/priority/priority.cc

```diff
@@ -170,6 +167,38 @@ class PriorityLb : public LoadBalancingPolicy {
       RefCountedPtr<ChildPriority> priority_;
     };
 
+    class DeactivationTimer : public InternallyRefCounted<DeactivationTimer> {
+     public:
+      explicit DeactivationTimer(RefCountedPtr<ChildPriority> child_priority);
+
+      void Orphan() override;
+
+     private:
+      static void OnTimer(void* arg, grpc_error_handle error);
+      void OnTimerLocked(grpc_error_handle);
+
+      RefCountedPtr<ChildPriority> child_priority_;
+      grpc_timer timer_;
+      grpc_closure on_timer_;
+      bool timer_pending_ = false;
```

Comment:
nit: can we just set `timer_pending_ = true` here, and below in the failover timer, to save a line in the ctor bodies of each? (I saw we're doing this in the weighted targets fix - https://github.com/grpc/grpc/pull/29203/files)

Reply:
Good catch; I meant to come back and make the same change here but forgot.  Done.